webapp/python/README.md

```diff
@@ -15,29 +20,54 @@
    * create environment `python3 -m venv ./venv`
    * activate it `source ./venv/bin/activate`
 3. Install required package dependencies: `pip install -r requirements.txt`
-    
+
 ## Run the example
 
 1. Start the Django development server: `python manage.py runserver 0.0.0.0:8000`
 2. Start your browser and make it point to the url `http://<host>:8000`
-   
-   To make it reachable by the Collabora Online server use as `<host>` the IP address of the machine where the Django 
-   server is running. In case the Django server can't be reached you could also need to open the port 8000 on the firewall. 
-   
+
+Or with HTTPS after configuring the certificates (see below)
+
+1. Start the Django development server: `run_https.sh`
+2. Start your browser and make it point to the url `https://<host>:8000`
```

Comment:
How will COOL allow talking with this server with a self-signed certificate.
I don't know myself.
We should probably mention something about it.

Reply:
I stashed my patche and did `make` && `make run` and it works. I think in debug it doesn't verify. That's the baseline. Also it's all `localhost`.